test/Microsoft.Framework.OptionsModel.Test/ConfigurationBinderExceptionTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Framework.ConfigurationModel;
+using Xunit;
+
+namespace Microsoft.Framework.OptionsModel.Test
+{
+    public class ConfigurationBinderExceptionTests
+    {
+        [Fact]
+        public void ExceptionWhenTryingToBindToInterface()
+        {
+            var input = new Dictionary<string, string>
+            {
+                {"ISomeInterfaceProperty:Subkey", "x"}
```

Comment:
I think the default VS formatting puts a space after the `{` and before the `}` no? If so, please format this so that the next person won't have a bunch of space-only changes.